var/spack/repos/builtin/packages/g4abla/package.py

```diff
@@ -11,19 +11,24 @@ class G4abla(Package):
     """Geant4 data for nuclear shell effects in INCL/ABLA hadronic mode"""
     homepage = "http://geant4.web.cern.ch"
     url = "http://geant4-data.web.cern.ch/geant4-data/datasets/G4ABLA.3.0.tar.gz"
+    
     maintainers = ['drbenmorgan']
 
-    version(
-        '3.0', sha256='99fd4dcc9b4949778f14ed8364088e45fa4ff3148b3ea36f9f3103241d277014')
-    version(
-        '3.1', sha256='7698b052b58bf1b9886beacdbd6af607adc1e099fc730ab6b21cf7f090c027ed')
+    # Only versions relevant to Geant4 releases built by spack are added
+    version('3.1', sha256='7698b052b58bf1b9886beacdbd6af607adc1e099fc730ab6b21cf7f090c027ed')
+    version('3.0', sha256='99fd4dcc9b4949778f14ed8364088e45fa4ff3148b3ea36f9f3103241d277014')
 
     def install(self, spec, prefix):
         mkdirp(join_path(prefix.share, 'data'))
         install_path = join_path(prefix.share, 'data', 'G4ABLA{0}'
                                  .format(self.version))
         install_tree(self.stage.source_path, install_path)
 
+    def setup_dependent_run_environment(self, env, dependent_spec):
+        install_path = join_path(prefix.share, 'data', 'G4ABLA{0}'
```

Comment:
Does this actually work?
Where is just `prefix` set?

Could you try this?
```suggestion
    def setup_dependent_run_environment(self, env, dependent_spec):
        install_path = join_path(self.prefix.share.data, 'G4ABLA{0}'
```

Reply:
Fixed in 0d4a05a